conn/physic/units.go

```diff
@@ -1389,10 +1364,11 @@ func dtoi(d decimal, scale int) (int64, error) {
 }
 
 // Converts a string to a decimal form. The return int is how many bytes of the
-// string are numeric. The string may contain +-0 prefixes and arbitrary
-// suffixes as trailing non number characters are ignored.
-// Significant digits are stored without leading or trailing zeros, rather an
-// exponent is used.
+// string are considered numeric. The string may contain +-0 prefixes and
+// arbitrary suffixes as trailing non number characters are ignored.
+// Significant digits are stored without leading or trailing zeros, rather a
```

Comment:
I don't understand what you mean with the first part, as it's stored as an int64 now.

Reply:
I'm trying to say 1000 is stored as base:1, exp:3.
0.001 stored as base: 1, exp: -3, 
and 12.34 as base: 1234 exp: -2

Reply:
... and that there is a limit to the number of significant digits (int64) eg you can't have 1234567890123456790.1234567890123456789.
But you can store very large or very small numbers 16* 10^-34 (0.00000000000000000000000000000000016) etc

Reply:
ahh now I understand.